src/locales/scripts/get-translations.js

```diff
@@ -144,11 +241,28 @@ if (process.argv.includes("--watch")) {
 }
 
 if (!process.argv.includes("--en-only")) {
-  const localeJSON = require("./wp-locales.json")
+  const username = process.env.GLOTPRESS_USERNAME
+  const password = process.env.GLOTPRESS_PASSWORD
+  if (username && password) {
+    console.log("Auth credentials found, performing bulk download.")
 
-  fetchAndConvertJed1xTranslations(Object.values(localeJSON).map((i) => i.slug))
-    .then((res) => {
-      console.log(`Successfully saved ${res.length + 1} translations.`)
-    })
-    .catch(console.error)
+    getAuthCookies(username, password)
+      .then(fetchBulkJed1xTranslations)
+      .then(extractZip)
+      .then((res) => {
+        console.log(`Successfully saved ${res.length + 1} translations.`)
+      })
+      .catch(console.error)
```

Comment:
```suggestion
      .catch(console.error)
```

This is a bit verbose, as it logs out the entire Axios result. A custom message withg the status code might be more legible. 

Should we also do the parallel download if the bulk fails instead?